release_files/Dockerfile.ubuntu

```diff
@@ -80,64 +81,60 @@ SHELL ["/bin/bash", "-c"]
 WORKDIR /
 
 COPY drivers /drivers
-
-RUN	set -e ; \
-	set -x ; \
-        apt update -y ; \
-        apt install -y curl libpugixml1v5 libtbb2 && \
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN	apt-get update -y ; \
+    apt-get install -y curl libpugixml1v5 libtbb2 --no-install-recommends && \
 	if [ "$GPU" == "1" ] ; then \
-                apt-get update && apt-get  install -y libnuma1 ocl-icd-libopencl1 && rm -rf /var/lib/apt/lists/* && \
+                apt-get update && apt-get  install -y libnuma1 ocl-icd-libopencl1 --no-install-recommends && rm -rf /var/lib/apt/lists/* && \
 	        case $INSTALL_DRIVER_VERSION in \
                 "20.35.17767") \
-                        mkdir /tmp/gpu_deps && cd /tmp/gpu_deps ; \
```

Comment:
Are you sure this will work? Below curls are not specifying /tmp/gpu_deps catalog as output so the installation with dpkg might not work.

Reply:
reverted to old method with `cd` and added hado ignore